### DISCUSSION TITLE
Upgrading to use AndroidX new Back APIs

### DISCUSSION THREAD

**Discussion by amlcurran** [software-mansion/react-native-screens#1789] [Ideas]
I've recently taken over an app which uses React Native, Navigation, and therefore Screens under the hood. 

Whilst testing on Android 14, the [predictive back animation](https://developer.android.com/guide/navigation/custom-back/predictive-back-gesture) is not supported and I believe it is down to the [version of Fragments](https://github.com/software-mansion/react-native-screens/blob/main/android/build.gradle#L126) Screens is using. 

I'd be interested in helping to update this library to support that and was wondering if that would be appreciated?

**Comment by tboba** [comment]
Hi @amlcurran! Speaking about AndroidX, we've been recently thinking about supporting predictive back gesture, so this was planned to implement but if you're willing to help by supporting us - please go ahead! Everyone are more than welcome to collaborate 🙏 

Cheers!

**Comment by SanjayDevTech** [comment]
Hi it's been a year, any updates on this one 🙂
Thanks
